evalml/tests/objective_tests/test_fraud_detection.py

```diff
@@ -121,8 +121,8 @@ def test_fraud_objective_score(X_y_binary):
     score = fraud_cost.score(y_true, out, extra_columns)
     assert (score == 0.0)
 
-    out = ww.DataColumn(fraud_cost.decision_function(y_predicted, 5, extra_columns))
-    pd.testing.assert_series_equal(out.to_series(), y_true, check_dtype=False, check_names=False)
+    out = pd.Series(fraud_cost.decision_function(y_predicted, 5, extra_columns))
```

Comment:
Is this necessary? I would assume the output of `decision_function` is already a pandas series

Reply:
It's not! Good catch